help/grandorgue.xml

```diff
@@ -9963,6 +9987,19 @@ played as is (without any loop/release handling)
      </para>
           </listitem>
         </varlistentry>
+        <varlistentry>
+          <term>Pipe999HasIndependentRelease</term>
+          <listitem>
+            <para>
+	      (boolean, default: rank HasIndependentRelease setting). If true
+	      then the release is played independently on the attack.
```

Comment:
If true, then releases are played independently of attacks in this pipe.

Reply:
@oleg68 Technically, this PR works beautifully in any test I've thrown at it. As soon as the help text is corrected I'll approve.

Reply:
Fixed